packages/@okta/vuepress-site/docs/guides/sign-into-mobile-app-redirect/main/ios/checkfortoken.md

```diff
@@ -0,0 +1,36 @@
+Check for an existing session in two steps.
+
+1. Initialize the authorization state manager (`authStateManager`) from a saved session.
+2. Verify that the access token is still valid by checking the expiry.
+
+The state manager session and its associated token are already saved by the `signIn` function calling `writeToSecureStorage`.
+
+Update the `tokenExpired` utility function to check for token expiry:
```

Comment:
Clarity and succinctness:

Check for an expired token by updating the `tokenExpired` utility function:

Reply:
updated